src/transport/raw/MessageHeader.cpp

```diff
@@ -158,11 +160,22 @@ CHIP_ERROR PacketHeader::Decode(const uint8_t * const data, uint16_t size, uint1
     err = reader.Read32(&mMessageId).StatusCode();
     SuccessOrExit(err);
 
+#ifdef DEBUG
+    ChipLogDetail(Inet, "PacketHeader::Decode");
+    ChipLogDetail(Inet, "  header=%u", (int) header);
+    ChipLogDetail(Inet, "  mEncryptionType=%d", (int) mEncryptionType);
+    ChipLogDetail(Inet, "  mFlags=%d", mFlags.Raw());
+    ChipLogDetail(Inet, "  mMessageId=%d", mMessageId);
+#endif
+
     if (mFlags.Has(Header::FlagValues::kSourceNodeIdPresent))
     {
         uint64_t sourceNodeId;
         err = reader.Read64(&sourceNodeId).StatusCode();
         SuccessOrExit(err);
+#ifdef DEBUG
```

Comment:
Again, why do we want this DEBUG thing?  Same throughout this file.  And again, all the logging in this file should be a separate PR.

Reply:
addressed in latest